Fixed Get-AzResourceGroup on -Id requires Set-AzContext to given subs… (#21765)
Azure/azure-powershell: src/Resources/Resources/ChangeLog.md

* Fixed `New-AzADApplication` when multiple redirect url types are provided. [#21108]
* Fixed `Update-AzADServicePrincipal` when empty array passed for `IdentifierUri` [#21345]
* Fixed an issue where location header was missing in the response from the service for New-AzManagedApplication.

> * Fixed `Get-AzResourceGroup` on -Id requires `Set-AzContext` to given subscription first [#21725]

Reviewer: ```suggestion
* Fixed `Get-AzResourceGroup` ignored the subscription ID in `-Id` [#21725]
```
